pkg/generators/markers.go

```diff
@@ -293,57 +296,150 @@ func lintArrayIndices(markerComments []string, prefix string) error {
 	return nil
 }
 
+// ExtractCommentTags parses comments for lines of the form:
+//
+//	'marker' + "key=value".
```

Comment:
can you give an example in the docs of input and output? We're changing the return type compared to `types.ExtractCommentTags` and it would be great to have that documented.

Reply:
Added